tools/Google.Cloud.ClientTesting/CodeHealthTester.cs

```diff
@@ -67,5 +67,33 @@ public static void AssertClassesAreSealedOrAbstract(Type sampleType, IEnumerable
             // Force output to show the bad types
             Assert.Equal(new string[0], badTypes.ToList());
         }
+
+        /// <summary>
+        /// Asserts that the assembly containing the given type does not define the disallowed namespaces.
+        /// </summary>
+        public static void AssertNoDisallowedNamespaces(Type sampleType, IEnumerable<string> disallowedNamespeces)
```

Comment:
If you make this a `params` array, you won't need the `new string[]` in the calling code.